### ISSUE TITLE
Quests are rolled back and can be completed again

### ISSUE THREAD

**Issue by Bonebrkr** [WarEmu/WarBugs#2278]
I had completed this quest once and received my Stalker Gloves. A few days later logged in and noticed that some quests in Highpass that I completed were available again (most noticeably was the ale quest). Checked my inventory and noticed my stalker gloves were not equipped anymore and also not in inventory. I went and checked and quest was available again so I took and completed stage 1 but the second part was never offered. Same on the ale quest, I went and found the cask near Bugs but after handing that in was not asked to do second part. Again I know this is Alpha and these things need to be worked out. Not sure if there is a way to make Chosen Poorly repeatable and if not I'm just SOL. Thanks for your hard work.

**Comment by Evildoor** [comment]
Do not post about lost items on bug tracker. Item loss is a known bug which has no solution so far. Dev policy is "no refunds" for now, so the only thing you can do is hope that when server restarts items will return. If they don't - tough luck, but that's how it stands.

**Comment by Bonebrkr** [comment]
It wasn't just about the lost item, the quest was available again but only the first part, when it was turned in second part was not available to be taken. I was told to report this on bug tracker, which is what I did. I was not sure if this was caused by a server roll back, although the " Head and Shoulders" quest which I did after Chosen was not affected. Just FYI, I am not expecting anything back, hence the "SOL"
